src/drivers/barometer/mpl3115a2/mpl3115a2.cpp

```diff
@@ -141,12 +136,12 @@ class MPL3115A2 : public cdev::CDev
 	/**
 	 * Initialize the automatic measurement state machine and start it.
 	 *
-	 * @param delay_ticks the number of queue ticks before executing the next cycle
+	 * @param delay_us the number of microseconds before executing the next cycle
 	 *
 	 * @note This function is called at open and error time.  It might make sense
 	 *       to make it more aggressive about resetting the bus in case of errors.
 	 */
-	void			start_cycle(unsigned delay_ticks = 1);
+	void			start_cycle(unsigned delay_us = 1);
```

Comment:
@dagar, most of the other drivers I checked simply name this method `start()`, this might be a good time to standardize the mpl3115a2 against the others.  `stop_cycle()` vs `stop()` is the same in this class.

Reply:
Changing to start/stop.